### DISCUSSION TITLE
Cannot deploy - page 42

### DISCUSSION THREAD

**Discussion by realmartinbaun** [symfony/symfony#48836] [Symfony Book: The Fast Track]
Hi

I have done every page of the book The fast track 6.0 but on page 42 I get an error.
When I write the command "symfony cloud:deploy" in the terminal (Windows 10) I get this error:

Are you sure you want to push to the main (production) branch? [Y/n] y

Pushing HEAD to the existing environment main
  command-line line 0: keyword certificatefile extra arguments at end of line
  fatal: Could not read from remote repository.

  Please make sure you have the correct access rights
  and the repository exists.

Please help me! - I'm new to Symfony and the other programs in the book so please dumb it down for me :)

**Comment by polekstulod** [comment]
@realmartinbaun Did you find the solution? Same problem here.

**Reply by realmartinbaun** [reply]
No, unfortunately
